test/urlcat.ts

```diff
@@ -3,79 +3,79 @@ import { expect } from 'chai';
 
 describe('urlcat', () => {
 
-  it ('Concatenates the base URL and the path if no params are passed', () => {
+  it('Concatenates the base URL and the path if no params are passed', () => {
     const expected = 'http://example.com/path';
     const actual = urlcat('http://example.com', 'path');
     expect(actual).to.equal(expected);
   });
 
-  it ('Uses exactly one slash for joining even if the base URL has a trailing slash', () => {
+  it('Uses exactly one slash for joining even if the base URL has a trailing slash', () => {
     const expected = 'http://example.com/path';
     const actual = urlcat('http://example.com/', 'path');
     expect(actual).to.equal(expected);
   });
 
-  it ('Uses exactly one slash for joining even if the path has a leading slash', () => {
+  it('Uses exactly one slash for joining even if the path has a leading slash', () => {
     const expected = 'http://example.com/path';
     const actual = urlcat('http://example.com', '/path');
     expect(actual).to.equal(expected);
   });
 
-  it ('Uses exactly one slash for joining even if the base URL and the path both have a slash at the boundary', () => {
+  it('Uses exactly one slash for joining even if the base URL and the path both have a slash at the boundary', () => {
     const expected = 'http://example.com/path';
     const actual = urlcat('http://example.com/', '/path');
     expect(actual).to.equal(expected);
   });
 
-  it ('Substitutes path parameters', () => {
+  it('Substitutes path parameters', () => {
     const expected = 'http://example.com/path/1';
     const actual = urlcat('http://example.com/', '/path/:p', { p: 1 });
     expect(actual).to.equal(expected);
   });
 
-  it ('Allows path parameters at the beginning of the path', () => {
+  it('Allows path parameters at the beginning of the path', () => {
     const expected = 'http://example.com/1';
     const actual = urlcat('http://example.com/', ':p', { p: 1 });
     expect(actual).to.equal(expected);
   });
 
-  it ('Parameters that are missing from the path become query parameters', () => {
+  it('Parameters that are missing from the path become query parameters', () => {
     const expected = 'http://example.com/path/1?q=2';
     const actual = urlcat('http://example.com/', '/path/:p', { p: 1, q: 2 });
     expect(actual).to.equal(expected);
   });
 
-  it ('Uses exactly one ? to join query parameters even if the path has a trailing question mark', () => {
-    const expected = 'http://example.com/path?q=2';
-    const actual = urlcat('http://example.com/', '/path?', { q: 2 });
+  it('Uses exactly one ? to join query parameters even if the path has a trailing question mark', () => {
+    const expected = 'http://example.com/path?q=true';
```

Comment:
Why did you change `p` to `true` here?

Reply:
ohh, sorry for that,I was just trying but forgot to delete.